backend/jobs/log-alerts/log-alerts.go

```diff
@@ -64,7 +72,9 @@ func WatchLogAlerts(ctx context.Context, DB *gorm.DB, TDB timeseries.DB, MailCli
 			func() error {
 				for range time.NewTicker(time.Duration(f) * time.Second).C {
 					alerts := (*alertsByFrequency)[f]
+					log.WithContext(ctx).Infof("Processing %d log alerts for frequency %d", len(alerts), f)
 					for _, alert := range alerts {
+						alert := alert
```

Comment:
added comment - iterators in golang use a single variable and update its value rather than copying it by value within the loop, which creates issues for goroutines:
https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables

Reply:
wow TIL. Thanks!